tools/azure-rest-api-specs-examples-automation/ci.yml

```diff
@@ -0,0 +1,32 @@
+trigger: none
+
+pr: none
+
+schedules:
```

Comment:
We have typically done the schedules on the DevOps side but I don't mind if you wnat to do this here as well.

Reply:
Sure. I removed `schedules` and `variables` from the ci.yml. Would configure them in DevOps.

Reply:
Probably outside of usual redmond and shanghai core business hours.